File: idea-plugin/src/main/java/org/ifinalframework/jetbrains/plugins/aio/api/yapi/model/Page.java
package org.ifinalframework.jetbrains.plugins.aio.api.yapi.model;


import lombok.Getter;
import lombok.Setter;

import java.util.List;


/**
 * Page
 *
 * @author iimik
 * @since 0.0.1
 **/
@Getter
@Setter
public class Page<T> {
    private Long total;
    private Long count;
    private List<T> list;
}

File: idea-plugin/src/main/kotlin/org/ifinalframework/plugins/jetbrains/aio/api/yapi/model/Page.kt
package org.ifinalframework.plugins.jetbrains.aio.api.yapi.model


/**
 * Page
 *
 * @author iimik
 * @since 0.0.1
 **/
data class Page<T>(
    val total: Long,
    val count: Long,
    val list: List<T>
)
